pkgs/applications/editors/emacs/elisp-packages/manual-packages/lsp-bridge/default.nix

```diff
@@ -88,7 +89,23 @@ melpaBuild {
     runHook postCheck
   '';
 
-  passthru.updateScript = unstableGitUpdater { };
+  passthru.updateScript = writeScript "update.sh" ''
+    #!/usr/bin/env nix-shell
+    #!nix-shell -i bash -p common-updater-scripts coreutils git
+    set -eu -o pipefail
+
+    tmpdir="$(mktemp -d)"
+    git clone --depth=1 https://github.com/manateelazycat/lsp-bridge.git "$tmpdir"
+
+    pushd "$tmpdir"
+    commit=$(git show -s --pretty='format:%H')
+    # Based on: https://github.com/melpa/melpa/blob/2d8716906a0c9e18d6c979d8450bf1d15dd785eb/package-build/package-build.el#L523-L533
+    version=$(git show -s --pretty='format:%cd' --date='format:%Y%m%d.%H%M')
```

Comment:
Thanks marsam!! ♥️ Just two things I noticed, the "decimal" part of the version needs to have the leading zeros removed, and timezone should be normalized to UTC:

I think this should fix those cases:
```shell
#!nix-shell -i bash -p common-updater-scripts coreutils git gnused
...
version=$(TZ=UTC git show -s --pretty='format:%cd' --date='format-local:%Y%m%d.%H%M' | sed 's|\.0*|.|')
```

Reply:
Good catch! I've updated the PR with your snippet, thanks!